cvat-ui/src/components/models-page/models-page.tsx

```diff
@@ -43,21 +56,20 @@ export default function ModelsPageComponent(props: Props): JSX.Element {
         );
     }
 
-    const uploadedModels = models.filter((model): boolean => model.id !== null);
-    const integratedModels = models.filter((model): boolean => model.id === null);
+    const [uploadedModels, integratedModels] = getModels(models);
 
     return (
         <div className='cvat-models-page'>
             <TopBarComponent installedAutoAnnotation={installedAutoAnnotation} />
-            { !!integratedModels.length
-                && <BuiltModelsList models={integratedModels} />
-            }
             { !!uploadedModels.length && (
-                <UploadedModelsList
-                    registeredUsers={registeredUsers}
-                    models={uploadedModels}
-                    deleteModel={deleteModel}
-                />
+                <>
+                    <BuiltModelsList models={integratedModels} />
```

Comment:
We do not need to render this component if ``integratedModels.length === 0``

Reply:
It will render if only **integratedModels.length** will be grater than 0

Reply:
But condition is ``uploadedModels.length``, not ``integratedModels.length``